datasets/catalonia_independence/catalonia_independence.py

```diff
@@ -0,0 +1,113 @@
+    language = "English",
+    ISBN = "979-10-95546-34-4",
+}
+"""
+
+_DESCRIPTION = """\
+This dataset contains two corpora in Spanish and Catalan that consist of annotated Twitter messages for automatic stance detection. The data was collected over 12 days during February and March of 2019 from tweets posted in Barcelona, and during September of 2018 from tweets posted in the town of Terrassa, Catalonia.
+
+Each corpus is annotated with three classes: AGAINST, FAVOR and NEUTRAL, which express the stance towards the target - independence of Catalonia.
+"""
+
+_HOMEPAGE = "https://github.com/ixa-ehu/catalonia-independence-corpus"
+
+_LICENSE = "CC BY-NC-SA 4.0"
+
+_URLs = {
+    "catalan": "https://github.com/ixa-ehu/catalonia-independence-corpus/raw/master/01_CIC_CA.zip",
+    "spanish": "https://github.com/ixa-ehu/catalonia-independence-corpus/raw/master/02_CIC_ES.zip",
+}
+
+
+class CataloniaIndependence(datasets.GeneratorBasedBuilder):
+    """This dataset contains two corpora in Spanish and Catalan that consist of annotated Twitter messages for automatic stance detection."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="catalan", version=VERSION, description="This part of my dataset covers a first domain"
+        ),
+        datasets.BuilderConfig(
+            name="spanish", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "catalan"
+
+    def _info(self):
+        features = datasets.Features(
+            {"id_str": datasets.Value("string"), "TWEET": datasets.Value("string"), "LABEL": datasets.Value("string")}
```

Comment:
can you use the ClassLabel Feature type for the LABEL field please ?
For example
```
"LABEL": datasets.ClassLabel(names=["AGAINST", "FAVOR", "NEUTRAL"])
```

you'll need to regenerate the dataset_infos.json file after this change

Reply:
thanks for the tip! will add it :)